prusti-viper/src/encoder/procedure_encoder.rs

```diff
@@ -2071,6 +2071,74 @@ impl<'p, 'v: 'p, 'tcx: 'v> ProcedureEncoder<'p, 'v, 'tcx> {
                             }
                         }
 
+                        "core::slice::<impl [T]>::len" => {
```

Comment:
Could you move the body of this case to a separate method?

Reply:
done.